src/components/reusable/errorComponent/errorComponent.scss

```diff
@@ -0,0 +1,49 @@
+@use '../../../common/scss/global.scss';
+@use '~@kyndryl-design-system/shidoka-foundation/scss/mixins/typography.scss';
+
+.error-component-container {
+
+    align-self: center;
+    display: grid;
+    place-items: center;
+
+    .error-image {}
+
+    .error-title {
+        h1 {
+            @include typography.type-headline-01;
+            font-weight: var(--kd-font-weight-bold);
```

Comment:
It looks like the title should be color: --kd-color-text-brand

Reply:
done.